app/services/vendor_status.rb

```diff
@@ -8,6 +10,16 @@ def initialize(from: nil, from_idv: nil, sp: nil)
   IAL2_VENDORS = %i[acuant lexisnexis_instant_verify lexisnexis_trueid].freeze
   ALL_VENDORS = (IAL2_VENDORS + %i[sms voice]).freeze
 
+  def vendor_outage_url_localized
+    return vendor_outage_url if I18n.locale == :en
+
+    vendor_outage_url(locale: I18n.locale)
+  end
+
+  def url_options
+    {}
+  end
```

Comment:
plan B version of https://github.com/18F/identity-idp/pull/5570/files#r740607678, make this return the locale param?

```suggestion
  include LocaleHelper

  def url_options
    { locale: locale_url_param }
  end
```